testing/functional/tests/cypress/integration/student/02-login-page.spec.ts

```diff
@@ -0,0 +1,139 @@
+import WelcomeObject from "../../page-objects/student-objects/WelcomeObject"
+import Login from "../../page-objects/student-objects/02-login-object"
+
+describe("Login Page", () => {
+
+    const Welcome = new WelcomeObject()
+    const LoginObject = new Login()
+
+    beforeEach(() => {
+        cy.visit("/")
+        cy.wait(2000)
+    })
+
+    it("Verify that user able to login with a valid username and valid password.", () => {
+        Welcome.virtualTestingBtn()
+        cy.fixture('testdata').then((testdata) => {
+            LoginObject.cardSerialNumberInputTxt().type(testdata.validCardSerialNumber).should('have.value', testdata.validCardSerialNumber)
+            LoginObject.cardSerialNumberContinuebtn()
+            LoginObject.passcodeInputTxt().type(testdata.validPasscode).should('have.value', testdata.validPasscode)
+            LoginObject.passcodeContinurBtn()
+            LoginObject.verifyLoggedInTxt()
+        })
+    })
+
+    it("Verify that user is not able to redirect to next page with invalid Card Serial Number in Virtual Card Testing.", () => {
+        Welcome.virtualTestingBtn()
+        cy.fixture('testdata').then((testdata) => {
+            LoginObject.cardSerialNumberInputTxt().type(testdata.invalidCardSerialNumber)
+            LoginObject.cardSerialNumberContinuebtn()
+            LoginObject.cardNotFoundTxt()
+        })
+    })
+
+    it("Test with empty Card Serial Number such that it must get failed.", () => {
+        Welcome.virtualTestingBtn()
+        cy.fixture('testdata').then((testdata) => {
+            LoginObject.cardSerialNumberInputTxt()
+            LoginObject.cardSerialNumberContinuebtn()
```

Comment:
Please ensure that the spell checker is up and running, it would detect issues like "Continuebtn".